Copyright 2024-2025 The Trivalent Authors

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software distributed under the License is
distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and limitations under the License.
---
diff --git a/chrome/browser/search_engine_choice/search_engine_choice_dialog_service_factory.cc b/chrome/browser/search_engine_choice/search_engine_choice_dialog_service_factory.cc
index e4d2eacf3f316..0dfdd8c0d365d 100644
--- a/chrome/browser/search_engine_choice/search_engine_choice_dialog_service_factory.cc
+++ b/chrome/browser/search_engine_choice/search_engine_choice_dialog_service_factory.cc
@@ -129,10 +129,6 @@ SearchEngineChoiceDialogServiceFactory::BuildServiceInstanceForBrowserContext(
 
   base::CommandLine* const command_line =
       base::CommandLine::ForCurrentProcess();
-  if (!g_is_chrome_build &&
-      !command_line->HasSwitch(switches::kForceSearchEngineChoiceScreen)) {
-    return nullptr;
-  }
 
   if (command_line->HasSwitch(switches::kNoFirstRun) &&
       !command_line->HasSwitch(
diff --git a/components/regional_capabilities/program_settings.cc b/components/regional_capabilities/program_settings.cc
index ad23174e4b..48867e9824 100644
--- a/components/regional_capabilities/program_settings.cc
+++ b/components/regional_capabilities/program_settings.cc
@@ -80,8 +80,7 @@ bool IsValidSerializedProgram(int serialized_program) {

 bool IsInProgramRegion(Program program,
                        const country_codes::CountryId& country_id) {
-  return base::Contains(GetSettingsForProgram(program).associated_countries,
-                        country_id);
+  return true;
 }

 bool IsClientCompatibleWithProgram(Program program) {
diff --git a/components/regional_capabilities/regional_capabilities_service.cc b/components/regional_capabilities/regional_capabilities_service.cc
index 4f95ae3d04..7f4d4953ec 100644
--- a/components/regional_capabilities/regional_capabilities_service.cc
+++ b/components/regional_capabilities/regional_capabilities_service.cc
@@ -169,7 +169,7 @@ Program CountryIdToProgram(const CountryId& country_id) {
   static constexpr Program kCountryDerivedPrograms[] = {
 #if BUILDFLAG(IS_IOS)
       // Only iOS can derive Taiyaki scope directly from the country.
-      Program::kTaiyaki,
+      //Program::kTaiyaki,
 #endif

       Program::kWaffle,
